### PR TITLE
This pull request to fix two translation issues

### PR DESCRIPTION
**Pull request by etienneCharignon**
Issue 1 : the "Modifier" links on the last page were not translated
Issue 2 : undefined value for the high suggestion for complementary health

### CODE REVIEW

File: source/components/conversation/FormDecorator.js [hunk 1]
```diff
@@ -32,7 +32,6 @@ export var FormDecorator = formType => RenderField =>
 				dispatch(change('conversation', field, value))
 		})
 	)
-	@translate()
```
Comment (Morendil) [inline source/components/conversation/FormDecorator.js]: This call is (usually) necessary when we want the component to re-render on switching translations, I'm not sure I understand how this fixes the problem you mentioned.

Comment (etienneCharignon) [reply source/components/conversation/FormDecorator.js]: For now, I don't understand how this is working. I've noticed that the translation does not append for the "modifier" link on the last page of the simulation if this @translate() is present in this file. I've checked that translations switching is still working even if it is not there. 

I wonder if this is working due to the fact that FormDecoractor is a sub part of something else already containing a @translate().

I agree that this needs more investigation.

**Comment by etienneCharignon** [comment]
I'm closing this PR to re-submit splited into parts

**Comment by Morendil** [comment]
Merci! C'est effectivement plus simple de gérer des PR plus atomiques.

(Je crois que sur syso on utilise le français pour les discussions sur les PR et les messages de commit… contrairement à beta.gouv.fr - it's all a bit confusing.)

**Pull request closed without merging**